### DISCUSSION TITLE
HTML5 compiling issue...

### DISCUSSION THREAD

**Discussion by TripleLLL9** [FunkinCrew/Funkin#1726] [Compiling help] [answered]
Hi. It's me again. Whenever I try to compile the HTML5 version of fnf, this happens:
<img width="610" alt="Another Compiling Issue" src="https://user-images.githubusercontent.com/86983398/132097245-7d69dd23-fff6-4a11-87c6-518addcbdefd.PNG">
Any more help?

**Comment by libauror-a** [comment, marked as answer]
those are warnings

**Reply by TripleLLL9** [reply]
ohhhhhhhhhhh.
